modules/ROOT/attachments/admin-api.yaml

```diff
@@ -1461,6 +1461,177 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/uuid'
+  /debug_bundle:
+    post:
+      tags:
+        - Debugging
+      summary: Start a debug bundle collection
+      operationId: post_debug_bundle
+      requestBody:
+        description: Request to start a debug bundle collection.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                job_id:
+                  type: string
+                  description: UUID for the debug bundle job.
+                  example: "123e4567-e89b-12d3-a456-426614174000"
+                config:
+                  type: object
+                  description: Optional configuration for debug bundle collection.
```

Comment:
It would be helpful to document the structure of config: https://github.com/redpanda-data/redpanda/blob/dev/src/v/redpanda/admin/api-doc/debug_bundle.json#L11-L109

Reply:
Why are SASL options required? Does the Redpanda cluster need to have SASL enabled before you can generate a bundle?

Reply:
> Why are SASL options required? Does the Redpanda cluster need to have SASL enabled before you can generate a bundle?

Authentication credentials are not required, SASL/SCRAM is supported, if specified, then all 3 fields are required.

Reply:
Thanks. Those fields appear required in the Console UI which is what confused me when I saw required in the spec.

https://redpandadata.slack.com/archives/C07DR90L940/p1729538029797419?thread_ts=1729521774.549549&channel=C07DR90L940&message_ts=1729538029.797419